tests/conftest.py

```diff
@@ -153,14 +155,16 @@ async def db():
 
 
 @pytest_asyncio.fixture
-async def insert_fake_resource():
-    async def _insert_fake_resource(database) -> None:
-        await database.execute(
-            f"""
-            INSERT INTO catalog (dataset_id, resource_id, url, priority, deleted)
-            VALUES ('{DATASET_ID}', '{RESOURCE_ID}', 'http://dev.local/', True, False)
-            ON CONFLICT (resource_id) DO NOTHING;
-            """
+async def insert_fake_resource(status: Union[str, None] = None):
+    async def _insert_fake_resource(database, status: Union[str, None] = None):
+        if status is None:
+            status = "TO_CHECK"
```

Comment:
I tried that earlier, but in that case it would mean that a `None` status can be passed to `Resource.insert`, so we would need a slight refacto of Resource.insert, which I didn't want to do in that PR.

Reply:
![Screenshot 2024-08-20 at 16 57 07](https://github.com/user-attachments/assets/2d6137cc-f92a-4a19-b0b8-d26fe096339d)

Reply:
ah yes i understand now, it makes sense 👍

Reply:
Actually, it would be totally OK with `async def _insert_fake_resource(database, status: str = "TO_CHECK"):`

Just fixed!